### ISSUE TITLE
Install Prerequisites

### ISSUE THREAD

**Issue by jzrlost** [dOpensource/dsiprouter#34]
I have attempted twice to install dsiprouter and must be missing something.  What is a "clean Debian" install?  I have loaded up Debian using the netinst ISO and installed nothing but ssh for remote access.  I then following the directions in the readme file and it gets through and now completes.  I did have to remove a "sudo" where it loads up the Kamailio key or it stops there with errors.  Getting past that it completes with some small errors about the rtpengine package not having kamailio dependency so it cannot configure it, but says it's all installed and ready.  I open a browser and try to get the login page it cannot be found.  Do I need to install Apache or some other web server?  Should I be installing is on a Debian LAMP server?  Thanks for your help, just need clarification on package prerequisites.

**Comment by mackhendricks** [comment]
Hi,

I will validate the install process.  What version of Debian were you using?  I usually use DigitalOcean to spin up Debian distro.  But, I can test it on a netinst ISO install.

**Comment by jzrlost** [comment]
I was using Debian Stretch.

Sent from my T-Mobile 4G LTE Device<div>
</div><div>
</div><div>-------- Original message --------</div><div>From: Mack Hendricks <notifications@github.com> </div><div>Date: 3/21/18  6:59 PM  (GMT-08:00) </div><div>To: dOpensource/dsiprouter <dsiprouter@noreply.github.com> </div><div>Cc: jzrlost <josh@netsteller.com>, Author <author@noreply.github.com> </div><div>Subject: Re: [dOpensource/dsiprouter] Install Prerequisites (#34) </div><div>
</div>Hi,

I will validate the install process.  What version of Debian were you using?  I usually use DigitalOcean to spin up Debian distro.  But, I can test it on a netinst ISO install.

**Comment by mackhendricks** [comment]
I fixed the master branch.  So, please do a pull and then:

/dsiprouter.sh uninstall
./dsiprouter.sh install

We are only supporting Stretch for this release due to the new framework we put in place for installation.

Sorry about the install issues.  Please let me know how it goes.  I also created a short video that shows the install process on Debian Stretch 9.4: https://youtu.be/Iu4BQkL1wGc

-Mack

**Comment by jzrlost** [comment]
Worked perfect.  Thank you for the quick fix.  We are going to do some test setups and see how it goes.  2 questions.
1. Does reloading Kamailio effect current calls (stateless or stateful)?
2. Is there an easy way to change the admin password?

**Comment by mackhendricks** [comment]
1) Reloading does not affect the current calls

2) Yes:

To change the configuration settings edit gui/settings.py file, e.g. vi ./gui/settings.py

USERNAME - web gui username
PASSWORD - web gui password
DSIP_PORT - port on which web gui is running, 5000 by default
DOMAIN - the domain used to create usernames for PBX and Endpoint registration.
You would then need to restart dSIPRouter with ./dsiprouter.sh restart

Note, we are moving all of the configurations to a database in upcoming releases.


Mack Hendricks / Head of Support / dOpenSource
web: http://dopensource.com <http://dopensource.com/>
support: +888-907-2085
dSIPRouter <http://dsiprouter.org/> - GUI focused on implementing Kamailio to provide SIP Trunking and PBX Hosting Services

**Comment by jzrlost** [comment]
Thank you for the quick reply.  I will check the settings above for the password update.
So far we have the asterisk server talking to the dsiprouter and the dsiprouter talking to the carrier.  We can make calls both inbound and outbound.  The only issue we are having is the audio is not there.  We did install dsiprouter with the -rtpengine. Not sure if it's some kind of NAT issue or not.
Setup:  Internet->router->dsiprouter->internal lan->asterisk.  We have forwarded ports 5060 and 10000-20000 from the router to dsiprouter.  Is that good or does dsiprouter have to be connected with a public IP?  We also only have 1 nic in the dsiprouter (virtual machine).
